docs/tutorials/google-smtp.rst

```diff
@@ -0,0 +1,42 @@
+Using Google Mail as an SMTP server
+===================================
+
+By default, Tutor comes with a simple SMTP server for sending emails. Such a server has an important limitation: it does not implement mailing good practices, such as DKIM or SPF. As a consequence. the emails you send might be flagged as spam by their recipients. Thus, you might want to disable the SMTP server and run your own, for instance using your Google Mail account.
+
+.. warning::
+  Google Mail SMTP servers come with their own set of limitations. For instance, you are limited to sending 500 emails a day. Reference: https://support.google.com/mail/answer/22839
+
+You should authorize third party to access your Google Mail account. In your Google Mail account, select "Manage Account", "Security", and turn on "Less Secure App Access".
```

Comment:
Can you add the link to the documentation for this step? 

https://support.google.com/accounts/answer/6010255 (as posted in the forum)

Reply:
done.